articles/cosmos-db/sql-api-sdk-node.md

```diff
@@ -35,10 +35,213 @@ ms.author: dech
+* Update to the latest Cosmos REST API version where all containers have unlimited scale
+* Make it easier to use Cosmos from the browser
+* Better align with the new Azure JS SDK guidlines
+
+#### Migration Guide for Breaking Changes
+##### Improved Client Constructor Options
+
+Constructor options have been simplified:
+
+* masterKey was renamed key and moved to the top-level
+* Properties previously under options.auth have moved to the top-level
+
+``` js
+// v2
+const client = new CosmosClient({
+    endpoint: "https://your-database.cosmos.azure.com",
+    auth: {
+        masterKey: "your-primary-key"
+    }
+})
+
+// v3
+const client = new CosmosClient({
+    endpoint: "https://your-database.cosmos.azure.com",
+    key: "your-primary-key"
+})
+```
+
+##### Simplified QueryIterator API
+In v2 there were many different ways to iterate or retrieve results from a query. We have attempted to simplify the v3 API and remove similar or duplciate APIs:
+
+* Remove iterator.next() and iterator.current(). Use fetchNext() to get pages of results.
+* Remove iterator.forEach(). Use async iterators instead.
+* iterator.executeNext() renamed to iterator.fetchNext()
+* iterator.toArray() renamed to iterator.fetchAll()
+* Pages are now proper Response objects intead of plain JS objects
+* const container = client.database(dbId).container(containerId)
+
+``` js
+// v2
+container.items.query('SELECT * from c').toArray()
+container.items.query('SELECT * from c').executeNext()
+container.items.query('SELECT * from c').forEach(({ body: item }) => { console.log(item.id) })
+
+// v3
+container.items.query('SELECT * from c').fetchAll()
+container.items.query('SELECT * from c').fetchNext()
+for await(const { result: item } in client.databases.readAll().getAsyncIterator()) {
+    console.log(item.id)
+}
+```
+
+##### Fixed Containers are now Paritioned
+The Cosmos service now supports partition keys on all containers, including those that were previously created as fixed containers. The v3 SDK updates to the latest API version that implements this change, but it is not breaking. If you do not supply a partition key for operations, we will default to a system key that works with all your existing containers and documents.
+
+##### upsert removed for Stored Procedures
+Previously upsert was allowed for non-partitioned collections, but with the API version update, all collections are partitioned so we removed it entirely.
+
+##### Item reads will not throw on 404
+const container = client.database(dbId).container(containerId)
+
+``` js
+// v2
+try {
+    container.items.read(id, undefined)
+} catch (e) {
+    if (e.code === 404) { console.log('item not found') }
+}
+
+// v3
+const { result: item }  = container.items.read(id, undefined)
+if (item === undefined) { console.log('item not found') }
+```
+
+##### Default Multi Region Write
+The SDK will now write to multiple regions by default if your database configuration supports it. This was previously opt-in behavior.
```

Comment:
Can we change "database configuration" -> Cosmos account configuration? Technically, whether multi-region write is supported is at the account level.

Reply:
Yep, updated